src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -13807,3 +13809,1451 @@ def test_aks_update_node_provisioning_profile(
             "aks delete -g {resource_group} -n {name} --yes --no-wait",
             checks=[self.is_empty()],
         )
+
+    # =========== start of valid localdns cases ===========
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix="clitest", location="westus2")
+    def test_aks_nodepool_add_with_localdns_config(self, resource_group, resource_group_location):
```

Comment:
Glad to see the enthusiasm for testing the new feature to ensure it works. However, the main purpose of scenario tests in azure-cli is to verify that the user's input is successfully sent to the server. To confirm whether the feature works as expected, it's better to test it in AKS E2E.

Could you keep only the most essential cases you added to help reduce the load on the test infrastructure? All scenario tests are run multiple times a day to check for regressions, but our test capacity is limited.